atest/interpreter.py

```diff
@@ -116,6 +134,19 @@ def classpath(self):
             return classpath + os.pathsep + tools_jar
         return tools_jar
 
+    @property
+    def java_opts(self):
+        if not self.is_jython:
+            return None
+        java_opts = os.environ.get('JAVA_OPTS')
+        if self.version_info[:3] >= (2, 7, 2) and self.java_version[0] >= 9:
+            if not java_opts:
+                java_opts = ''
```

Comment:
The default could be given to `os.environ.get()`. If the idea is to return `None` with Jython if not 2.7.2 on Java 9+, then it would be good to do that explicitly. I guess returning an empty string in that case is fine, though.

Reply:
The idea was to append `--add-opens java.base/java.io=ALL-UNNAMED --add-opens java.base/sun.nio.ch=ALL-UNNAMED` in case `JAVA_OPTS` contains something already given by user, or only set in case it is empty (or not set).

Reply:
I meant that instead of
```python
java_opts = os.environ.get('JAVA_OPTS')
if not java_opts_
    java_opts = ''
```
we could just have
```python
java_opts = os.environ.get('JAVA_OPTS', '')
```
Appending `--add-opens` to possibly existing `JAVA_OPTS` is a good idea.

Reply:
Yeah. I did that in this way. Please, have a look at last commit.